### PR TITLE
[DO NOT MERGE] 	split sample and library into different modules

### PR DESCRIPTION
**Pull request by FabianTerhorst**
- split sample and library into different modules
- update okhttp and lombok

Todo:
readd protobuf commit
fix build errors caused from incompatible proto files

### CODE REVIEW

File: src/resources/protobuf [hunk 1]
```diff
@@ -1 +0,0 @@
-Subproject commit ff99b436a34f0db79e3f1db805811f083405aecd
```
Comment (FabianTerhorst) [inline src/resources/protobuf]: i don´t know why this is happening

File: library/src/resources/protobuf [hunk 2]
```diff
@@ -0,0 +1 @@
+Subproject commit ff99b436a34f0db79e3f1db805811f083405aecd
```
Comment (FabianTerhorst) [inline library/src/resources/protobuf]: The submodule isn´t correctly added

**Comment by Grover-c13** [comment]
I added another example btw :p

**Comment by FabianTerhorst** [comment]
@Grover-c13 Then problem is that i can´t figure out how to reinit the submodule after changing the path.

**Comment by FabianTerhorst** [comment]
Maybe the easiest way to merge this would be recreate the branch.

**Comment by Grover-c13** [comment]
yeah maybe :P

**Pull request closed without merging**